predicators/envs/kitchen.py

```diff
@@ -146,7 +143,8 @@ def simulate(self, state: State, action: Action) -> State:
 
     def step(self, action: Action) -> Observation:
         self._gym_env.step(action.arr)
-        self._gym_env.render()
+        if self._using_gui:
```

Comment:
Did we not already have a flag for rendering in other environments?

Reply:
yeah we do -- this is using existing functionality. we can now use `--use_gui` and that toggles `self._using_gui` in the base env class